app/(afterLogin)/(okrplanning)/okr/settings/check-in-rule/check-in/index.tsx

```
@@ -0,0 +1,948 @@
+    useGetAllFeedbackRecords();
+  const { data: workSchedulesData } = useGetWorkSchedules();
+  const { tenantId } = useAuthenticationStore();
+  const queryClient = useQueryClient();
+  const [selectedCategoryId, setSelectedCategoryId] = useState<
+    string | undefined
+  >();
+  const [ruleType, setRuleType] = useState<
+    'time-based' | 'achievement-based' | 'both'
+  >('time-based');
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [selectedWorkScheduleId, setSelectedWorkScheduleId] = useState<string | undefined>();
```

ESLint

[error] 'selectedWorkScheduleId' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)